messages/en.json

```diff
@@ -41,11 +41,11 @@
   "admin/reviews.cancel": "Cancel",
   "admin/reviews.table.empty-state.approved": "There are no approved reviews.",
   "admin/reviews.table.applyMatch.error.0": "Unspecified error. Try again.",
-  "admin/reviews.table.applyMatch.error.partial.delete.plural": "{errorQuant} reviews haven't been deleted due to an unknown error",
+  "admin/reviews.table.applyMatch.error.partial.delete.plural": "{errorQuant, plural, one {# review hasn't been deleted} other {# reviews haven't been deleted}} due to an unknown error",
   "admin/reviews.table.applyMatch.error.partial.delete.singular": "{errorQuant} review hasn't been deleted due to an unknown error",
-  "admin/reviews.table.applyMatch.error.partial.approve.plural": "{errorQuant} reviews haven't been approved due to an unknown error",
+  "admin/reviews.table.applyMatch.error.partial.approve.plural": "{errorQuant, plural, one {# review hasn't been approved} other {# reviews haven't been approved}} due to an unknown error",
```

Comment:
Hi Cristian, it's because the pluralized string is not formatted to fit the [ICU standard](https://crowdin.com/blog/2022/04/13/icu-guide) and therefore can't be adapted to all the languages we translate into. If we don't do this transformation, we can't have languages like Polish, which has 4 different plurals, translated correctly depending on the number indicated by the variable.